From 5d191dac7d468c4af8644a18b8932036d9a6dad6 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Wed, 18 Nov 2020 01:47:31 -0800
Subject: [PATCH] defconfig: Don't sync when entering suspend

Syncing can take a long time in some cases, and the risk of us losing
power or crashing while in suspend on a battery-backed device is very
low. Android also enters and exits suspend very frequently, which
exacerbates the cost.

Change-Id: I35b9cb3e79532e18b2240b377b20708eae86cec2
Signed-off-by: Sevenrock <sevenrock@hotmail.de>
---
 arch/arm64/configs/vendor/rhodep_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/vendor/rhodep_defconfig b/arch/arm64/configs/vendor/rhodep_defconfig
index 2eb220b09743..d49de427c5c6 100644
--- a/arch/arm64/configs/vendor/rhodep_defconfig
+++ b/arch/arm64/configs/vendor/rhodep_defconfig
@@ -75,6 +75,7 @@ CONFIG_CMDLINE_EXTEND=y
 CONFIG_BUILD_ARM64_UNCOMPRESSED_KERNEL=y
 CONFIG_BUILD_ARM64_DT_OVERLAY=y
 CONFIG_TLB_CONF_HANDLER=y
+CONFIG_SUSPEND_SKIP_SYNC=y
 CONFIG_PM_WAKELOCKS=y
 CONFIG_PM_WAKELOCKS_LIMIT=0
 # CONFIG_PM_WAKELOCKS_GC is not set
-- 
2.39.2

